request_tracing.go

```diff
@@ -29,8 +29,12 @@ func init() {
 fetching values, and since the request-id is in a const, it is tied to this package. */
 // GetRequestID gets the RequestID from a context.context
 func GetRequestID(ctx context.Context) string {
+	if ctx == nil {
+		return ``
+	}
+
 	if ctx.Value(ctxRequestIDKey) == nil {
-		return ""
+		return ``
```

Comment:
I suggest we don't use the backtick quotes unless we have reason to, it seems to be the most common practice, and this might surprise future code readers.